.husky/typesVerification.sh

```diff
@@ -0,0 +1,28 @@
+echo "Running Types.ts generation test"
+
+
+TYPES_FILE="packages/client/hmi-client/src/types/Types.ts"
+
+MODIFIED_FILES=`git status -s | grep "Types.ts" | wc -l`
+if (( MODIFIED_FILES == 0 )); then
```

Comment:
Sorry for my lack of knowledge here.
Does this mean I will need to generate for every PR? Even if it has no java changes?

Reply:
I cant read